DOCUMENTATION.md

```diff
@@ -90,30 +148,30 @@ Your headers are clean ? Column `lemma` needs to stay `lemma` for your output so
     <key>lemma</key>
     <key>form</key>
 </header>
-``` 
+```
 
 This example will ignore `POS` column if there was one.
 
-##### When your headers differ
+#### When your headers differ
 
 Say you have two files, whose headers are: `form;lemma;POS` and `token;lemma;POS`. Well, we got you covered: if you want
-to map `token` to `form`, simply use the `map-to` attribute: 
- 
+to map `token` to `form`, simply use the `map-to` attribute:
+
 ```xml
 <corpora>
     <corpus path="./1.csv" column_marker=";">
         <header type="explicit">
             <key>lemma</key>
             <key>form</key>
             <key>POS</key>
-        </header>        
+        </header>
     </corpus>
     <corpus path="./2.csv" column_marker=";">
         <header type="explicit">
             <key>lemma</key>
             <key map-to="form">token</key>
```

Comment:
Add a comment ? `<!-- key=token is transformed into the map-to value "form" -->`?

Reply:
done

Reply:
Je bougerai le commentaire à l'endroit ou tu as `key>token</key`

Reply:
done